process/block/shardblock.go

```diff
@@ -921,6 +923,38 @@ func (sp *shardProcessor) updateState(headers []data.HeaderHandler) {
 	}
 }
 
+func (sp *shardProcessor) snapShotEpochStartFromMeta(header *block.Header) {
+	accounts := sp.accountsDB[state.UserAccountsState]
+	if !accounts.IsPruningEnabled() {
+		return
+	}
+
+	if header.IsStartOfEpochBlock() {
+		epochStartId := core.EpochStartIdentifier(header.GetEpoch())
+		metaBlock, err := process.GetMetaHeaderFromStorage([]byte(epochStartId), sp.marshalizer, sp.store)
+		if err != nil {
+			log.Warn("could not find epoch start metablock for", "epoch", header.GetEpoch(), "err", err)
```

Comment:
log.Debug? No validator cares about this warn.

Reply:
I would leave as log.Warn. it should never happen, only if there was something incorrectly coded.

Reply:
ok